scripts/machine-setup.ps1

```diff
@@ -18,8 +17,8 @@ if (!(Get-Command python -ErrorAction SilentlyContinue ))
 }
 
 & python -m pip --version
-& python -m pip install setuptools
-& python -m pip install glad
+& python -m pip install conan
+& conan config install https://github.com/novelrt/ConanConfig.git
```

Comment:
If this is being called in build.ps1, it should not be called here.

Reply:
Yeah, on second thought, for consistency this should be removed since Linux _only_ calls it in the `Generate` function of the build script.

Reply:
uhh, wait, so what goes where here?

Reply:
Just line 20 - remove the `conan config install` line since we're calling it in the build scripts already :)